packages/vue/src/composables/table.ts

```diff
@@ -166,7 +166,7 @@ export function useTable<
 		/** Deselects records on the current page. */
 		deselectPage: () => bulk.deselect(...table.value.records.map((record: RecordType) => getRecordKey(record))),
 		/** Whether all records on the current page are selected. */
-		isPageSelected: computed(() => table.value.records.every((record: RecordType) => bulk.selected(getRecordKey(record)))),
+		isPageSelected: computed(() => table.value.records.length > 0 && table.value.records.every((record: RecordType) => bulk.selected(getRecordKey(record)))),
```

Comment:
```suggestion
		isPageSelected: computed(() => ! table.value.records.some((record: RecordType) => ! bulk.selected(getRecordKey(record)))),
```

Reply:
Seems that you want to return false if you find any that you don't find selected instead

Reply:
yes